web_monitoring/diffing_server.py

```diff
@@ -74,16 +74,23 @@ def __init__(self, url, body, headers):
 
 access_control_allow_origin_header = \
     os.environ.get('ACCESS_CONTROL_ALLOW_ORIGIN_HEADER')
-
+ALLOWED_ORIGINS = set()
+if access_control_allow_origin_header:
```

Comment:
This feels like a lot of logic to just have sitting bare in the module. It might be nicer if this was packaged into a function and that function was called either in `make_app()` or lazily in `BaseHandler.set_default_headers()` (in both cases, you could cache the resulting set in `app.settings`).

Reply:
Good point, I moved the logic inside `BaseHandler.set_default_headers()`.